### ISSUE TITLE
The initialization of Category Embedding

### ISSUE THREAD

**Issue by LyapunovStability** [mims-harvard/UniTS#12]
Hello:
This is excellent work! Thank you for sharing it. Can I confirm if the category embedding is randomly initialized and fully trainable during training? For zero-shot classification, I believe it is initialized and fixed for continuously incoming samples.

Additionally, have you experimented with irregular time series data? I think it is interesting to explore if UnitTS also performs well in MIMIC-III or Physionet2012 datasets.

**Comment by gasvn** [comment]
Yes, in current version, the category embedding is randomly initialized.  If you want to do zero-shot classifiction, you can gather a few sample from each class and get the everaged embedding as the category embedding.

We haven't working on the irregular time series data. We will try it in our future work. Thank you!
